web/src/components/njwds-extended/cta/SingleCtaLink.tsx

```diff
@@ -20,7 +20,7 @@ export const SingleCtaLink = (props: Props): ReactElement => {
 
   return (
     <CtaContainer noBackgroundColor={props?.noBackgroundColor || false}>
-      <ActionBarLayout reverseInMobile={false} alignLeft={props?.alignLeft || false}>
+      <ActionBarLayout disableReverseOrderInMobile alignLeft={props?.alignLeft || false}>
```

Comment:
[dust] Any particular reason we don't want to explicitly call out the value as `true` here for readability's sake?

Reply:
Less code can prevent the code from wrapping to a new line. If we want to enforce a value, can we add an eslint rule?